lib/plug/ssl.ex

```diff
@@ -197,6 +268,36 @@ defmodule Plug.SSL do
     |> Keyword.put_new(:reuse_sessions, true)
   end
 
+  defp configure_managed_tls(options) do
+    case Keyword.get(options, :cipher_suite) do
```

Comment:
We can use `Keyword.pop/2` here. This way we don't need to call `Keyword.delete` in three different places anymore. :)